scalyr_agent/builtin_monitors/docker_monitor.py

```diff
@@ -1074,6 +1279,19 @@ def _initialize( self ):
 
         metrics_only = self._config.get('metrics_only')
 
+        self.label_exclude_globs = self._config.get( 'label_exclude_globs' )
+        self.label_include_globs = self._config.get( 'label_include_globs' )
+
+        if not scalyr_util.is_list_of_strings( self.label_include_globs):
```

Comment:
Why not move this into some sort of validate method on `DockerOptions`?

Reply:
The only reason is that validating a list of strings is something that I'm sure I've used elsewhere in the code (but I couldn't find which monitor), and is likely to be used in other config options in other monitors in the future.

Happy to move it in to DockerOptions for now, and move it back out if it turns out to be needed elsewhere later.

Reply:
Actually, upon further inspection, the call to this is happening in the `_initialize` method before any DockerOptions are created (they are created in the ContainerChecker at a different point in the code, so I'm not sure if DockerOptions is the best place to put it - otherwise we'll have to start passing another docker_options option around.

Currently `_initialize` ensures that the config items are valid, and then later the ContainerChecker is passed the config object and it reads the previously validated values and creates the DockerOptions.

I think leaving it as is for the moment is probably a better option - though happy to discuss (or have you override).

Reply:
Ok, that sounds fine to me.